src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -282,7 +282,7 @@ export default function SavedObjectFactory(esAdmin, kbnIndex, Promise, Private,
           if (_.get(err, 'origError.status') === 409) {
             const confirmMessage = `Are you sure you want to overwrite ${this.title}?`;
 
-            return safeConfirm(confirmMessage)
+            return confirmModalPromise(confirmMessage, { confirmButtonText: 'Overwrite' })
```

Comment:
Wherever the return value wasn't being used I used confirmModal. Since the promise is being returned here, I kept confirmModalPromise around.  I could wrap it, but there is another place that is returning the promise, so I would have been duplicating code at that point.  I didn't dig that deep however into how the returned promises were being used and whether they could extracted out.